bndtools.core/src/bndtools/model/repo/RepositoryResourceElement.java

```diff
@@ -41,6 +42,20 @@ public Resource getResource() {
 		return resource;
 	}
 
+	@SuppressWarnings("unchecked")
+	@Override
+	public <T> T getAdapter(Class<T> adapter) {
+
+		// this is triggered e.g. when you doubleclick on an
```

Comment:
In general we do not provide this comment. This is about a completely different part of the code and things might change in the future that invalidates this comment. The IAdaptable is the contract and that is documented in the code. It is fine to add this extra information in the commit.

I admit of not commenting public APIs in our code but not commenting non-public API is an explicit strategy in bnd.

Reply:
no hurry